pom.xml

```diff
@@ -71,11 +71,14 @@
     <!-- ################################################################################ -->
     <version.avalon-framework>4.1.4</version.avalon-framework>
     <version.batik>1.6.1-1</version.batik>
-    <version.com.ahome-it.lienzo-core>2.0.192-RC1</version.com.ahome-it.lienzo-core>
+    <version.com.ahome-it.lienzo-core>2.0.270-RELEASE</version.com.ahome-it.lienzo-core>
+    <version.com.ahome-it.lienzo-tests>1.0.0-RC1</version.com.ahome-it.lienzo-tests>
     <version.com.android-dx>1.10</version.com.android-dx>
     <version.com.github.detro>1.2.0</version.com.github.detro>
     <version.com.github.tomakehurst.wiremock>1.53</version.com.github.tomakehurst.wiremock>
     <version.com.google.android>4.1.1.4</version.com.google.android>
+    <!-- Override the version in ip-bom until https://github.com/jboss-integration/jboss-integration-platform-bom/pull/310 is merged -->
+    <version.com.google.gwt.gwtmockito>1.1.6</version.com.google.gwt.gwtmockito>
```

Comment:
This override should not be necessary anymore as we are now using ip-bom 7.0.0CR3 which defines gwtmockito in version 1.1.6.